client/README.md

```diff
@@ -29,11 +29,7 @@ npm run action client/src/www/start
 
 The latter command will open a browser instance running the app. Browser platform development will use fake servers to test successful and unsuccessful connections.
 
-The app logic is located in [src/www/app](src/www/app). UI components are located in [src/www/ui_components](src/www/ui_components). If you want to work specifically on an individual UI element, try the storybook!:
-
-```sh
-npm run action client/src/www/storybook
-```
+The app logic is located in [src/www/app](src/www/app). UI components are located in [src/www/ui_components](src/www/ui_components).
```

Comment:
I would maybe keep the instructions for storybook in here. This is where developers will be most likely to be looking for these types of instructions.